shell/protoc.sh

```diff
@@ -16,86 +11,166 @@ source "$SCRIPTS_DIR/lib/bootstrap.sh"
 
 PROTO_DOCS_DIR="$(get_repo_directory)/apidocs/proto"
 
-if [[ -n $CIRCLECI ]]; then
-  {
-    echo "warning: running protobuf generation in CI is only supported for bootstrap and is DEPRECATED"
-    echo "         this will result in only Go protobuf artifacts being generated"
-  } >&2
-  exec protoc --go_out=plugins=grpc,paths=source_relative:. --proto_path "$(get_repo_directory)/api" "$(get_repo_directory)/api/"*.proto
-fi
+PROTOC_IMPORTS_BASE_PATH="$HOME/.outreach/.protoc-imports"
```

Comment:
Hmm... are there any chances for conflict between protoc versions if they have the same dir?

Reply:
im thinking about it and I don't know if it matters to version this path with the protoc version too. i think it would matter if protoc was compiling the imports separately or something, but it isn't - it's just taking the proto files at face value at whatever revision you specify for that import

Reply:
Could we make it work with a random tempdir path?  Which would guarantee a unique path each time.